app/views/administrate/application/_collection.html.erb

```diff
@@ -45,23 +45,21 @@ to display a collection of resources in an HTML table.
           <% end %>
         </th>
       <% end %>
-      <% [valid_action?(:edit, collection_presenter.resource_name),
-          valid_action?(:destroy, collection_presenter.resource_name)].count(true).times do %>
-        <th scope="col"></th>
-      <% end %>
```

Comment:
I'm glad you simplified this; no need to make things complex when we can render simple blank columns.